libs/ui/layout/src/lib/truncated-text/truncated-text.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="relative">
+  <div class="flex items-center justify-between w-full gap-2">
+    <span #textElement class="truncate">
+      {{ text }}
+    </span>
+    <gn-ui-button
+      *ngIf="isTextTruncated"
+      [matTooltip]="'show.all.text' | translate"
+      matTooltipPosition="above"
```

Comment:
Minor but I wouldn't use matTooltip here; simple `title` attributes are fine and accessible